l10n_jp_summary_invoice/models/account_billing.py

```diff
@@ -82,25 +82,33 @@ def _compute_billing_date_due(self):
         "currency_id",
     )
     def _compute_tax_totals(self):
```

Comment:
@AungKoKoLin1997 I've refactored this method as the last part of the added test (i.e., `self.assertEqual(abs(billing_tax_amount), 20)`) would fail with the previous logic.

Reply:
I reviewed the code.
LGTM